src/screens/CreatePost/index.js

```diff
@@ -717,7 +715,7 @@
       animatedReminder.setValue(0);
     }
   }, [typeUser]);
-  console.log({typeUser}, 'heja');
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing the console log statement improves the cleanliness of the code and reduces unnecessary output during execution. Good job!


<!-- This is an auto-generated comment by OSS CodeRabbit -->